libraries/apollo-runtime/src/jsMain/kotlin/com/apollographql/apollo3/network/http/KtorHttpEngine.kt

```diff
@@ -27,7 +27,7 @@ actual class DefaultHttpEngine constructor(private val connectTimeoutMillis: Lon
     expectSuccess = false
     install(HttpTimeout) {
       this.connectTimeoutMillis = this@DefaultHttpEngine.connectTimeoutMillis
-      this.socketTimeoutMillis = this@DefaultHttpEngine.readTimeoutMillis
+      this.requestTimeoutMillis = this@DefaultHttpEngine.readTimeoutMillis
```

Comment:
Can we document what the different options do here? Also make sure the comment [here](https://github.com/apollographql/apollo-kotlin/blob/f1d5fd10730b8c011da83d17e1580600d29374c3/libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/network/http/HttpEngine.kt#L33) is still up to date?